hw/lesson-6/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+      <link href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" rel="stylesheet">
+      <title>Document</title>
+   </head>
+   <body>
+      <section class="wrapper">
+         <div class="blockDistance">
+            <div class="change">
+            <div class="icon_wrapper_1">
+              <i class="fas fa-tablet-alt"></i>
+            </div>
+         </div>
+            <h2>Responsive Design</h2>
+            <p>
+              Etiam sagittis velit vitae varius pellentesque. Suspendisse posuer condimentum velit, id vehicula nisi. 
+            </p>
+         </div>
+         <div class="blockDistance">
+            <div class="icon_wrapper_2">
```

Comment:
use tha same class instead of icon_wrapper_1, 2, 3. simple icon wrapper because they are all similar, we should reuse our classes